src/components/ResultTile.js

```diff
@@ -53,11 +54,15 @@ class ResultTile extends React.Component {
                     />
                 }
             >
-                <div>{object.name}</div>
-                <div>{object.site.name}</div>
+                <h2 className="name">{object.name}</h2>
+                <div className="site">{object.site.name}</div>
```

Comment:
Can you merge this with master? The class here is site_name, which is being used in a test. Happy to have either, so long as the test is correct.